### ISSUE TITLE
Allow application name to be set on non-authenticated requests.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#12]
```
The build() method does not have a user_agent property, so user_agent cannot be 
set on non-authenticated requests (The OAuth2 client does allow the user_agent 
to be set as part of the flow).
```

Original issue reported on code.google.com by `mon...@google.com` on 9 Mar 2011 at 6:59

**Comment by GoogleCodeExporter** [comment]
```
That title should read "*Allow* application name to be set..."
```

Original comment by `mon...@google.com` on 9 Mar 2011 at 7:00

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 9 Mar 2011 at 7:13
- Changed title: **Allow application name to be set on non-authenticated requests.**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision 7d4b772381.
```

Original comment by `jcgregorio@google.com` on 12 Mar 2011 at 12:26
- Changed state: **Fixed**
